test.lv.tf

```diff
@@ -43,4 +43,36 @@ resource "cloudflare_record" "mx_test_lv" {
   value   = "mail.kid.lv."
   type    = "MX"
   ttl     = 86400
-}
\ No newline at end of file
+}
+
+resource "cloudflare_record" "ns1_cadmium_test_lv" {
+  zone_id = cloudflare_zone.test_lv.id
+  name    = "cadmium"
+  value   = "ns-1382.awsdns-44.org"
+  type    = "NS"
+  ttl     = 1
```

Comment:
I think ttl=1 is for "proxied" resources in CF. for normal ones pick 86400

Reply:
Nice catch!